src/letsbuilda/pypi/models/models_json.py

```diff
@@ -114,6 +114,8 @@ class Info:
     version: str
     yanked: bool
     yanked_reason: str | None
+    license_expression: str | None
+    license_files: str | None
```

Comment:
```suggestion
    license_files: list[str] | None
```
https://github.com/pypi/warehouse/pull/16949/files#diff-730d8e751957f97fab26f9b37db88c03b058ad3d009e6a15de85cf14f587f608